doc/api/cli.md

```diff
@@ -1554,6 +1559,11 @@ is being linked to Node.js. Sharing the OpenSSL configuration may have unwanted
 implications and it is recommended to use a configuration section specific to
 Node.js which is `nodejs_conf` and is default when this option is not used.
 
+### `--env-file-optional=config`
+
+Behaviour is the same as [`--env-file`][], but an error is not thrown if the file
```

Comment:
Do we prefer UK English or US. `Behaviour` is the UK spelling, but don't mind changing it

Reply:
US: https://github.com/nodejs/node/blob/main/doc/README.md#documentation-style-guide

Reply:
Perfect, thanks! Will go ahead and change it as soon as I have time